projects/managed-hubs/support.md

```diff
@@ -181,25 +181,32 @@ When a new ticket lands in Freshdesk under the support group and it is not an in
 
 1. `Who: Support steward`
 
-   **Respond within 24 working hours**. Acknowledge receipt of the support request and let the {term}`Community Representative` know about any investigation we have done thus far.
+   **Respond within 24 working hours**. Acknowledge receipt of the support request and let the {term}`Community Representative` know a time-boxed investigation will start soon. Please request any additional information you may need to be able to reproduce the issue in step 2.
+
 2. `Who: Support steward`
 
    **Spend 30 minutes trying to resolve**. If you believe you can resolve the issue within 30 minutes, try resolving it yourself.
    1. If you resolve the issue, then jump to the "Confirm resolution" step 7.
    2. If you don't believe you can resolve the issue (or you couldn't) in 30 minutes, jump to the next step.
-3. `Who: Support Steward, the board metadata will be revisited by a Partnerships representative and the Engineering Manager`
+
+3. `Who: Support Steward`
 
    **Open an engineering issue**. If this is a {term}`Change Request` or {term}`Guidance Request` and/or you cannot resolve the issue within 30 minutes, then open a support issue for the team to discuss.
 
    [{bdg-primary}`Open a general issue`](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=&template=01_new-issue.yml) that describes the request and any next steps we should take. It should be automatically added to the **Eng & Prod** board by the existing automation. Add the **type**: `support` and choose an **impact** level in the **Eng & Prod** board metadata (available for the very same issue, no need to visit the board itself!).
    
    If the issue has a `critical` impact (we defer that first evaluation to the support steward), an additional ping to the support Slack channel is needed to boost the signal.
```

Comment:
Can you cross-link to a definition of "critical"? I think without this people will have different ideas of what "critical" means. It is probably similar to "what is an incident" but we should make that explicit and clarify when something can be critical even if not an incident

Reply:
Tried it at https://github.com/2i2c-org/team-compass/pull/764#discussion_r1315099927.
Waiting for feedback from folks about it.